pom.xml

```diff
@@ -85,27 +100,27 @@
       <dependency>
         <groupId>io.reactivex.rxjava2</groupId>
         <artifactId>rxjava</artifactId>
-        <version>2.2.21</version>
+        <version>${rxjava.version}</version>
```

Comment:
out of curiosity: why do we have RX Java AND the Reactor BOM?

Reply:
`rxjava` is used in the examples and `reactor-bom` provides versions for `reactor-core` and `reactor-test` that are used in this project. I started working on moving away from bom to latest versions to fully match the server, but there are some methods that were removed and that work needs more time.